docs/src/test/java/docs/http/javadsl/server/BlockingInHttpExamples.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.http.javadsl.server;
+
+import akka.actor.ActorSystem;
+import akka.dispatch.MessageDispatcher;
+import akka.http.javadsl.model.HttpResponse;
+import akka.http.javadsl.server.Route;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+import org.junit.Test;
+
+import java.util.concurrent.CompletableFuture;
+
+public class BlockingInHttpExamples extends JUnitRouteTest {
+
+    @Test
+    public void compileOnlySpec() throws Exception {
+        // just making sure for it to be really compiled / run even if empty
+    }
+
+    void blockingHttpDefaultDispatcher() {
+        //#blocking-example-in-default-dispatcher
+        // BAD (due to blocking in Future, on default dispatcher)
+        final Route routes = post( () ->
+                completeWithFuture(CompletableFuture.supplyAsync(() -> { // uses defaultDispatcher
```

Comment:
I added a short note on it :) d30ee16f6604e93e792c5b0fc65c83b18e21c9bf
Thanks for your efforts here! Merged 👍